great_expectations/expectations/core/expect_column_values_to_be_in_type_list.py

```diff
@@ -105,6 +105,7 @@ class ExpectColumnValuesToBeInTypeList(ColumnMapExpectation):
         [expect_column_values_to_be_of_type](https://greatexpectations.io/expectations/expect_column_values_to_be_of_type)
     """
 
+    condition_parser: Union[str, None] = "pandas"
```

Comment:
Not necessary here but do we have a select number of `condition_parsers`? We should probably use a `StrEnum` or `Literal[...]` as a follow-up ticket